main.py

```diff
@@ -424,10 +424,8 @@ def get_short_info(github):
     else:
         string += "> 🚫 " + translate["Not Opted to Hire"] + "\n > \n"
 
-    string += '> 📜 ' + translate['public repositories'] % public_repo + " " + '\n > \n' if public_repo > 1 else translate[
-        'public repository'] % public_repo + ' \n > \n'
-    string += '> 🔑 ' + translate['private repositories'] % private_repo + " " +' \n\n' if private_repo > 1 else translate[
-        'private repository'] % public_repo + ' \n > \n'
+    string += '> 📜 ' + translate['public repositories'] % public_repo + " " + '\n > \n'
+    string += '> 🔑 ' + translate['private repositories'] % private_repo + " " +' \n > \n'

```

Comment:
What about something like this?

```suggestion
    string += '> 📜 ' 
    string += translate['public repositories'] % public_repo + " " + '\n > \n' if public_repo != 1 else translate[`public repository`] % public_repo + ` \n > \n`
    string += '> 🔑 ' 
    string += translate['private repositories'] % private_repo + " " +' \n > \n' if private_repo != 1 else translate[`private repository`] % private_repo + ` \n > \n`
```

This fixes the 3 main problems:
1. update `public_repo` to `private_repo` in `else` block for private repository count
2. always print icons, regardless of `if` or  `else`
3. refactor verbiage to `0 ... Repositories`, `1 ... Repository`, and `n ... Repositories`

But, as always, open for any more feedback. 😄

Reply:
Ya opened your PR will wait for more feedback on this.

Reply:
> Ya opened your PR will wait for more feedback on this.

Oh great, I'd much rather be able to directly use this repository than my fork. 🥇 

Based on your feedback, I have since made a few changes,

https://github.com/anmol098/waka-readme-stats/blob/a5eacaa9ae5e469c673f4559ae7c98d3122b0f05/main.py#L427-L430

Please let me know if you have any other feedback!